### ISSUE TITLE
Update argus-docker defaults when a new Argus backend with changed SPA environment variables is released

### ISSUE THREAD

**Issue by lunkwill42** [Uninett/argus-docker#12]
Once Uninett/Argus#907 is merged and released in a new Argus version (whose version number should be noted here):  Once argus-docker is updated to use this version of Argus by default, we must also take care to update the names of the renamed environment variables (the ones used to control how the Argus backend interacts with the frontend SPA)

**Comment by lunkwill42** [comment]
Check state of #17 in regards to this

**Comment by lunkwill42** [comment]
This has become stale - it was fixed a long time ago, and is no longer relevant under Argus 2.
